### ISSUE TITLE
1.4.0 release links not working

### ISSUE THREAD

**Issue by mrcnski** [paritytech/polkadot-sdk#2550]
The format for downloading the releases described [here](https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-polkadot#installing-the-polkadot-binaries) seems to have changed. For example, this works:

https://github.com/paritytech/polkadot-sdk/releases/download/polkadot-v1.1.0/polkadot

But the same format doesn't work for the latest release:

https://github.com/paritytech/polkadot-sdk/releases/download/polkadot-v1.4.0/polkadot

**Comment by chevdor** [comment]
Good catch, thanks for reporting this issue. I fixed it, your 2nd link now works.

**Comment by mrcnski** [comment]
Thanks @chevdor, much appreciated! Just to confirm, the format has also been fixed for the worker binaries? Thanks!
